schemas/kb.py

```diff
@@ -0,0 +1,81 @@
+"""
+Knowledge Base Schema
+
+This is a very general schema that covers many information extraction-style
+tasks, including:
+
+- Named entity recognition (NER)
+- Named entity disambiguation/canonicalization/normalization (NED)
+- Event extraction
+- Relation extraction (RE)
+- Coreference resolution
+
+This schema assumes a document with child elements (e.g., entities, relations)
+and a flat hierarchy of document passages.
+
+"""
+import datasets
+
+
+features = datasets.Features(
+    {
+        "id": datasets.Value("string"),
+        "document_id": datasets.Value("string"),
+        "passages": datasets.Sequence(
+            {
+                "id": datasets.Value("string"),
+                "type": datasets.Value("string"),
+                "text": datasets.Value("string"),
+                "offsets": datasets.Sequence([datasets.Value("int32")]),
+            }
+        ),
+        "entities": datasets.Sequence(
+            {
+                "id": datasets.Value("string"),
+                "offsets": datasets.Sequence([datasets.Value("int32")]),
+                "text": datasets.Sequence([datasets.Value("string")]),
```

Comment:
I think this should be just `Sequence(Value("string"))` not a nested iterable

Reply:
This is to match the discontinuous offset definition. The ugliness of supporting discontinuous spans necessitates this unfortunately.

Reply:
Even for the discontinuous case, a nested set of offsets `[(0,4), (9,12)]` leads to one less level of nesting for the text right? `["text1", "text2"]`